monai/transforms/lazy/utils.py

```diff
@@ -129,6 +129,6 @@ def resample(data: torch.Tensor, matrix: NdarrayOrTensor, spatial_size, kwargs:
         "padding_mode": kwargs.pop(LazyAttr.PADDING_MODE, None),
     }
     resampler = monai.transforms.SpatialResample(**init_kwargs)
-    # resampler.lazy_evaluation = False
-    with resampler.trace_transform(False):  # don't track this transform in `data`
+    # resampler.lazy_evaluation = False  # resampler is a lazytransform
```

Comment:
Hi @wyli @yiheng-wang-nv ,

May I know why keep this line commented code?

Thanks.

Reply:
Hi @Nic-Ma , `.lazy_evaluation` requires the transform extends `LazyTransform`, now the class in dev branch does not have related code (see: https://github.com/Project-MONAI/MONAI/pull/5860/files#diff-3a1b9fb96f9cde878a33605fcbd39cf1af51a79c285fe662d80a00f0c43e2d5fL114) This line will be uncommented after related changes are added.